compiler/rustc_session/src/options.rs

```diff
@@ -1317,7 +1317,10 @@ options! {
         "control generation of position-independent code (PIC) \
         (`rustc --print relocation-models` for details)"),
     remark: Passes = (Passes::Some(Vec::new()), parse_passes, [UNTRACKED],
-        "print remarks for these optimization passes (space separated, or \"all\")"),
+        "output remarks for these optimization passes (space separated, or \"all\")"),
+    remark_dir: Option<PathBuf> = (None, parse_opt_pathbuf, [UNTRACKED],
```

Comment:
Shouldn't this be a `-Z` option until we decide to stabilize it?

Reply:
Probably yes, thanks! I wasn't sure how stabilization of `rustc` flags works.

Reply:
Would comma-separated be possible? Space-separated requires using quotes in the shell, which is a bit ugly.

Reply:
This is pre-existing code, and the flag is already stabilized, I'm not sure if we can change it. Most of the time you'll want just "all" anyway, I think.